README.md

```diff
@@ -105,12 +105,13 @@ This test suite expects following environment variables set:
  - `STORAGE_API_MAINTENANCE_URL` - URL for maintenance testing (https://maintenance-testing.keboola.com/)
 
 
-You can export variables manually or you can create and fill file `set-env.sh` as copy of attached `set-env.template.sh`.
+You can export variables manually, or you can create and fill file `set-env.sh` as copy of attached `set-env.template.sh`.
 
-Than  you can run tests:
+You can obtain these values by calling REST API described in [Confluence](https://keboola.atlassian.net/wiki/spaces/TECH/pages/1145307137/Testing+init+commands)
```

Comment:
Repo je verejne, takze v podstate kazdy zakaznik je schopny ho zacat pouzivat a vidi ho, ale prave na nas private atlassian sa nikto okrem nas nedostane. Imho by to malo sluzit ako tak obecny navod a nemal by tu byt odkaz na nejake nase interne veci. Ale mozno na dokumentaciu je tu master @tomasfejfar tak mozno na to mal iny nazor :)

Reply:
To je legitimní poznámka. Nemám na to principielně názor. Pro zprovoznění by mělo stačit vyplnit ENVy a jejich názvy jsou docela popisné. A když chybí třeba feature, tak si to taky řekne. 

Takže by to tam možná fakt být nemělo.

Reply:
pravda, opraveno